docs/topics/http/sessions.txt

```diff
@@ -72,25 +72,39 @@ If you have multiple caches defined in :setting:`CACHES`, Django will use the
 default cache. To use another cache, set :setting:`SESSION_CACHE_ALIAS` to the
 name of that cache.
 
-Once your cache is configured, you've got two choices for how to store data in
-the cache:
+Once your cache is configured, you have to choose between a database-backed
+cache or a non-persistent cache.
 
-* Set :setting:`SESSION_ENGINE` to
-  ``"django.contrib.sessions.backends.cache"`` for a simple caching session
-  store. Session data will be stored directly in your cache. However, session
-  data may not be persistent: cached data can be evicted if the cache fills
-  up or if the cache server is restarted.
+The non-persistent cache ``django.contrib.sessions.backends.cache`` is faster
+because it disregards database persistence. But you may have to handle edge
+cases caused by the cache being ephemeral. This might lead to a more complex
```

Comment:
Ephemeral. Is that a well recognised term for folk using this? Even as a native English speaker I had to Google the meaning of it.

Reply:
You're right. Thanks. I'll try to find a better way of explaining this. Will rebase and push again to keep this at one commit.

Reply:
@smithdc1 I changed `ephemeral` to `non-persistent` or `unreliable, transient`

I think these should get the point we're trying to convey across. Thanks for the feedback.